css/templates/lune/components/_sidebar.css

```diff
@@ -0,0 +1,99 @@
+@import "../page-vars";
+
+.ampstart-sidebar {
+  width: 80vw;
+  /* Keep the header bar visible when the sidebar is open */
+  height: calc(100vh - var(--layout-header-height-mobile));
```

Comment:
According to the template's design, the top header is always visible, even with the sidebar open. The hamburger icon in the header both opens and closes the sidebar, and animated between a hamburger icon and a close icon when the sidebar toggles.

We'll tag you in the design deck, so that you can better understand the requirements.

I don't think there is a need for a new component, it's just a matter of either being able to stack the header on top of the sidebar (in terms of `z-index`), or for the sidebar to sit below the header (which is what I did in this PR).